src/web/app/pom.xml

```diff
@@ -327,6 +327,7 @@
           -->
           <!-- Make sure it's not looking for annotations everywhere -->
           <contextXml>${project.build.testOutputDirectory}/jetty-context.xml</contextXml>
+          <useTestScope>true</useTestScope>
```

Comment:
I never use ``jetty:run``, anyone else does? Otherwise I'll trust you on this one :-D
Personally I always start GeoServer from the IDE, using Start.java in the web/app module.

Reply:
I have IDE issues. :)